package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "reastaurant-page for Microverse",
+  "main": "webpack.config.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
## Add missing script
- [ ] Kindly add back the `start` script here.

Reply:
ok. done I did return back the start scrip with the:  
```
"scripts": { "start": "webpack serve --config webpack.dev.js", 
 "build": "webpack --config webpack.prod.js"
 },
```